lib/contextMenu.js

```diff
@@ -173,6 +173,33 @@ export default function (self) {
         self.parentDOMNode.appendChild(downArrow);
       }
     }
+    function fade(element) {
+      var op = 1;
+      var timer = setInterval(function () {
```

Comment:
If `timer` is not used anywhere, do we still need it?

Reply:
timer is used for clearInterval

Reply:
But is `clearInterval` used? I don't see it.

Reply:
![Screenshot_2](https://user-images.githubusercontent.com/78894564/144705028-bf2ce837-d769-4ee6-821f-0b58f277e67d.png)

Reply:
Oh, wow — that's embarrassing 🤦 

Apologies!